### ISSUE TITLE
proper multi-language support

### ISSUE THREAD

**Issue by sdumetz** [Holusion/eCorpus#50]
currently, the `?lang=XX` queryString is generally ignored in favor of `setups[0].language`

However it's not clear for users how this property is changed. 

Furthermore, it should probably be set to the user's language on creation to prevent defaults being filled-in from some default value later-on.

TODO:
- identify where exactly  `setups[0].language` is set
- find a better way to set it right
- Verify reproductibility

**Comment by sdumetz** [comment]
`setups[0].language` is serialized in **CVSetup.cacheSetupState()** from **CVLanguageManager.toData()**, which currently reads:

```
language: ELanguageType[ins.language.getValidatedValue()] as TLanguageType,
```

It means we get whatever's the active language when the scene's state is captured.

It links the inconsistent behaviour to the bizarre way scenes are serialized with two-steps save, the `setup`only being save if it was captured.

This warrants a quick test where we set `setups[0].language` on document creation, then see if it still tends to get switched-over over time.

Otherwise, unless we still have loading race conditions in language management, that might be it for this issue, leaving the whole problem to UX design troubleshooting.

**Comment by sdumetz** [comment]
Will be fixed with https://github.com/Smithsonian/dpo-voyager/pull/280

**Comment by sdumetz** [comment]
Fixed in release v0.42.0
